auth-service-2/app/main/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 def create_app(config_name):
 
-    print("start again again  ...............more")
+    print("start again again  ...............more more")
```

Comment:
use python logger class (if you can) and (use logger.info) for information logging